android/app/src/main/java/org/fossasia/openevent/activities/auth/UserProfileActivity.java

```diff
@@ -68,9 +65,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
 
+        userProfileActivityViewModel = ViewModelProviders.of(this).get(UserProfileActivityViewModel.class);
         if (!AuthUtil.isUserLoggedIn()) {
             //Delete User data from realm
-            RealmDataRepository.getDefaultInstance().clearUserData();
+            userProfileActivityViewModel.clearUserDataFromRealm();
```

Comment:
Don't use implementation specific names. We will migrate from Realm soon. So it should be agnostic